aten/src/THNN/generic/SpatialUpSamplingBicubic.c

```diff
@@ -9,7 +9,7 @@ void THNN_(SpatialUpSamplingBicubic_updateOutput)(
     int output_height,
     int output_width,
     bool align_corners) {
-  AT_ERROR('This function is deprecated, please use it from ATen.');
+  THError('This function is deprecated, please use it from ATen.');
```

Comment:
This has nothing to do with `AT_ERROR` versus `THError`. Your actual problem is you need to use a double quote, not a single quote.

Reply:
you're right! my fault. I have figured it out. I have fixed that locally. thanks!